qark/plugins/Dynamically_Loading_Code.py

```diff
@@ -5,25 +5,58 @@
 from modules import common
 from lib.pubsub import pub
 import lib.plyj.parser as plyj
+import lib.plyj.model as m
 
 
 class DynamicallyLoadingCodePlugin(IPlugin):
     DEX_CLASS_LOADER = r'DexClassLoader'
     CLASS_LOADER = r'loadClass'
     DYNAMIC_BROADCAST_RECEIVER = r'registerReceiver'
+    global list_BR
+    list_BR = []
```

Comment:
Sometimes the same issue can be found in different files. Therefore, we need a list to collect all the file paths containing the issue and display it all at once rather than overwhelming the output screen with the same issue description for every different file.